packages/components/ng-ovh-sidebar-menu/src/list/item/factory.js

```diff
@@ -589,6 +589,15 @@ export default /* @ngInject */ function($q, $timeout) {
     return self;
   };
 
+  SidebarMenuListItem.prototype.isValidNamespace = function isValidNamespace(
+    namespace,
+  ) {
+    if (Array.isArray(this.namespace)) {
```

Comment:
maybe
```js
return namespace === undefined || this.namespace.includes(namespace);
```
and remove the undefined values from sidebar.constants.js

Reply:
We want to have some items on default namespace (`undefined`), some on another one (`hpc`) and some on both.
I can update sidebar config to add all on `server` namespace instead of `undefined` but it could be a BC.